docs/README.md

```diff
@@ -21,6 +21,8 @@ installed. Also install the following libraries:
     # Following is needed only for generating API docs
     $ sudo pip install sphinx
     $ sudo Rscript -e 'install.packages(c("knitr", "devtools", "roxygen2", "testthat"), repos="http://cran.stat.ucla.edu/")'
+    $ sudo Rscript -e 'install.packages(c("rmarkdown"), repos="http://cran.stat.ucla.edu/")'
+    $ sudo pip install pandoc pandoc-citeproc
```

Comment:
@felixcheung i agree, if we are installing manually on Ubuntu/Debain we need to install using below command
``sudo apt-get install pandoc pandoc-citeproc``

similarly for
Fedora/Red Hat: ``sudo yum install pandoc``
Arch: ``sudo pacman -S pandoc``
Mac OS X with Homebrew: ``brew install pandoc pandoc-citeproc Caskroom/cask/mactex``
Machine with Haskell: ``cabal-install pandoc``

@srowen  
TTBOMK, as it's python package, it can be manage via pip also
https://pypi.python.org/pypi/pypandoc